clang/include/clang/Basic/AArch64SVEACLETypes.def

```diff
@@ -200,7 +200,7 @@ SVE_PREDICATE_TYPE_ALL("__clang_svboolx4_t", "svboolx4_t", SveBoolx4, SveBoolx4T
 
 SVE_OPAQUE_TYPE("__SVCount_t", "__SVCount_t", SveCount, SveCountTy)
 
-AARCH64_VECTOR_TYPE_MFLOAT("__MFloat8_t", "__MFloat8_t", MFloat8, MFloat8Ty, 1, 8, 1)
+AARCH64_VECTOR_TYPE_MFLOAT("__mfp8", "__u6mfp8", MFloat8, MFloat8Ty, 1, 8, 1)
```

Comment:
From the usage in https://github.com/llvm/llvm-project/blob/9a1a8f33850d6c7a090bbe7fcf60dcdda93482d0/clang/lib/AST/ItaniumMangle.cpp#L3437 )
it looks to me it  needs to be either (preferable)

    AARCH64_VECTOR_TYPE_MFLOAT("__mfp8", "__mfp8", MFloat8, MFloat8Ty, 1, 8, 1)

or

    AARCH64_VECTOR_TYPE_MFLOAT("__mfp8", "u6__mfp8", MFloat8, MFloat8Ty, 1, 8, 1)